docs/framework/wpf/data/data-binding-how-to-topics.md

```diff
@@ -1,63 +0,0 @@
----
```

Comment:
I actually refer to this article from the desktop guide. As the desktop guide isn't completed, there is a lot of information here with specific tutorials. I think we should keep this until I get the majority ported.